packages/vue-dot/src/elements/PageContainer/PageContainer.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<VSheet
+		class="vd-page-container"
+		:class="paddingClass"
+		:width="containerSize"
```

Comment:
```suggestion
		:width="containerSize"
		:class="paddingClass"
		class="vd-page-container"
```
Ce genre d'éléments devrait être géré par le linter mais on met les attributs dynamiques avant les attributs statiques